backend/gateway/mux/pprof.go

```diff
@@ -0,0 +1,20 @@
+package mux
+
+import (
+	"net/http"
+	"net/http/pprof"
+)
+
+func pprofHandlers(httpMux *http.ServeMux) {
+	httpMux.Handle("/debug/pprof/", http.HandlerFunc(pprof.Index))
+	httpMux.Handle("/debug/pprof", http.HandlerFunc(pprof.Index))
+	httpMux.Handle("/debug/pprof/cmdline", http.HandlerFunc(pprof.Cmdline))
+	httpMux.Handle("/debug/pprof/profile", http.HandlerFunc(pprof.Profile))
+	httpMux.Handle("/debug/pprof/symbol", http.HandlerFunc(pprof.Symbol))
+	httpMux.Handle("/debug/pprof/trace", http.HandlerFunc(pprof.Trace))
+	httpMux.Handle("/debug/goroutine", pprof.Handler("goroutine"))
+	httpMux.Handle("/debug/heap", pprof.Handler("heap"))
```

Comment:
index already handles the other profiles from what i can tell 
https://github.com/golang/go/blob/master/src/net/http/pprof/pprof.go#L380-L396
https://github.com/golang/go/blob/dda2991c2ea0c5914714469c4defc2562a907230/src/runtime/pprof/pprof.go#L186-L193

Reply:
let me give it a try